queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-01-01';
+SELECT * FROM animals WHERE neutered = true AND escape_attempts < 3;
```

Comment:
- [ ] Again here we are supposed to select the name and not `select * from` as you have done, kindly fix this as well.

requirements screenshot
![image](https://github.com/dieum512/Vet-clinic/assets/81932478/c97717e3-dbfa-4104-8bb9-91c3c9dc50a4)